doc/conf.py

```diff
@@ -15,6 +15,9 @@
 from __future__ import print_function
 import sys
 import os
+
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
Many ways. For example you could use the ![develop mode][https://pythonhosted.org/setuptools/setuptools.html#develop-deploy-the-project-source-in-development-mode] or change your ``PYTHONPATH``.
I don't have an "installed" version of scikit-learn, I only have the repo checkout ;)